RandomError1.py

```diff
@@ -0,0 +1,226 @@
+            Pscore+=.12589
+        if ASCII is '+':
+            Pscore+=.10000
+        if ASCII is ',':
+            Pscore+=.07043
+        if ASCII is '-':
+            Pscore+=.06310
+        if ASCII is '.':
+            Pscore+=.05012
+        if ASCII is '/':
+            Pscore+=.03981
+        if ASCII is '0':
+            Pscore+=.03162
+        if ASCII is '1':
+            Pscore+=.02512
+        if ASCII is '2':
+            Pscore+=.01995
+        if ASCII is '3':
+            Pscore+=.01585
+        if ASCII is '4':
+            Pscore+=.01259
+        if ASCII is '5':
+            Pscore+=.01000
+        if ASCII is '6':
+            Pscore+=.00794
+        if ASCII is '7':
+            Pscore+=.00631
+        if ASCII is '8':
+            Pscore+=.00501
+        if ASCII is '9':
+            Pscore+=.00398
+        if ASCII is ':':
+            Pscore+=.00316
+        if ASCII is ';':
+            Pscore+=.00251
+        if ASCII is '<':
+            Pscore+=.00200
+        if ASCII is '=':
+            Pscore+=.00158
+        if ASCII is '>':
+            Pscore+=.00126
+        if ASCII is '?':
+            Pscore+=.00100
+        if ASCII is '@':
+            Pscore+=.00079
+        if ASCII is 'A':
+            Pscore+=.00063
+        if ASCII is 'B':
+            Pscore+=.00050
+        if ASCII is 'C':
+            Pscore+=.00040
+        if ASCII is 'D':
+            Pscore+=.00032
+        if ASCII is 'E':
+            Pscore+=.00025
+        if ASCII is 'F':
+            Pscore+=.00020
+        if ASCII is 'G':
+            Pscore+=.00016
+        if ASCII is 'H':
+            Pscore+=.00013
+        if ASCII is 'I':
+            Pscore+=.00010
+        if ASCII is 'J':
+            Pscore+=.00008
+    Pscorelist.append(Pscore)
+    return Pscorelist
+
+def phredscore(fastq_fh):
+    Pscorelist=[]
+    for line in fastq_fh:
+        if 'B' in line:
+            Pscorelist=phredvalues_version8(line)
+            return Pscorelist
+
+
+
+
+
+
+def errorproducer(fastq_fh):
+    errorcount=0
+    totaltotalbase=0
+    Pscoreplacemark=0
+    dankhold=0
+    Pscorelist=[]
+    for line in fastq_fh:
+        if dankhold==0:
+            Pscorelist=phredscore(fastq_fh)
+        dankhold=1
+        totalbase=0
+        placemark=0
+        newstring=""
+        line=line.rstrip()
+        #getting rid of \'s
+        if 'B' in line:
+            continue
+        if '@' in line:
+            continue
+        if '}' in line:
+            continue
+        if '+' in line:
+            continue
+        if '-' in line:
+            continue
+        if ';' in line:
+            continue
+        #really sloppily discounting any lines apart from those with basepairs
+        totalbase=len(line)
+        totaltotalbase+=totalbase
+        Pscoreplacemark=1
+        print Pscorelist
+        for basepair in line:
+            placemark+=1
+            errorchance=Pscorelist[Pscoreplacemark-1]
+            int(errorchance)
```

Comment:
I'm fairly certain you have to do something like this for this to work:

errorchance = int(errorchance)